connectors/aws/aws-sns/element-templates/aws-sns-inbound-boundary.json

```diff
@@ -4,7 +4,7 @@
   "id": "io.camunda.connectors.inbound.AWSSNS.Boundary.v1",
   "description": "Receive events from AWS SNS",
   "version": 2,
-  "documentationRef": "https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/aws-sns-inbound/",
+  "documentationRef": "https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/amazon-sns/?amazonsns=inbound/",
```

Comment:
@Oleksiivanov You might need to remove the pending slash. For me it leads to opening the page with the outbound tab selected: https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/amazon-sns/?amazonsns=inbound/